app/src/main/res/values/themes.xml

```diff
@@ -47,6 +47,7 @@
         <item name="colorOnSurfaceInverse">@color/md_theme_light_inverseOnSurface</item>
```

Comment:
Please change line 21 to

`<resources xmlns:tools="http://schemas.android.com/tools">`

Otherwise the file is invalid because we use the tools namespace in your item.